server/src/test/java/org/elasticsearch/cluster/ESAllocationTestCase.java

```diff
@@ -144,11 +217,6 @@ public Decision canAllocate(ShardRouting shardRouting, RoutingNode node, Routing
         public Decision canAllocate(ShardRouting shardRouting, RoutingAllocation allocation) {
             return decision;
         }
-
-        @Override
-        public Decision canAllocate(RoutingNode node, RoutingAllocation allocation) {
```

Comment:
why is this removed? it is not on the referred ES state https://github.com/elastic/elasticsearch/blame/704317da71c3de1b85679bc549bba3b4230acb5d/test/framework/src/main/java/org/elasticsearch/cluster/ESAllocationTestCase.java#L228
Seems like this could result in wrong behaviour.

Reply:
Good pick, i took it from the wrong point-of-time (head). My mistake. Thanks.

Reply:
I merged it already in the commit.